src/go/k8s/tests/e2e/create-topic-with-client-auth/01-create-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rpk-config
+data:
+  redpanda.yaml: |
+    redpanda:
+    rpk:
+      tls:
+        key_file: /etc/tls/certs/tls.key
+        cert_file: /etc/tls/certs/tls.crt 
+        truststore_file: /etc/tls/certs/ca.crt
```

Comment:
I'm not sure if `require_client_auth` defaults to true? 

NIT: I would set this explicitly.

REF:
```
redpanda:
  kafka_api_tls:
    enabled: true
    require_client_auth: true
    cert_file: <path to PEM-formatted cert file>
    key_file: <path to PEM-formatted key file>
    truststore_file: <path to PEM-formatted truststore file>
```

Reply:
yeah this is kind of confusing but actually, the client_auth is just server setting while this config is used by RPK and that's not a server, so that setting actually would not be interpreted and does not make a difference

Reply:
Ok thanks for explanation.